packages/block-library/src/image/image-editor.js

```diff
@@ -259,6 +265,29 @@ export default function ImageEditor( {
 
 	return (
 		<>
+			{ ! inProgress && (
+				<div
+					className="richimage__zoom-control"
+					aria-label={ __( 'Zoom' ) }
+				>
+					<Icon icon={ search } />
+					<RangeControl
+						min={ MIN_ZOOM }
+						max={ MAX_ZOOM }
+						value={ Math.round( zoom ) }
+						onChange={ setZoom }
+					/>
+					<ToolbarItem>
```

Comment:
Shouldn't this only be used within a `<Toolbar>`?

Reply:
I suppose yes, I'll look for an alternative.